### PR TITLE
Added option to launch robot in simulator mode

### PR DESCRIPTION
**Pull request by newtoncn**
Just a little update to allow me to easily switch between simulator launches and real robot launches.

### CODE REVIEW

**Comment by newtoncn** [comment]
@garethellis0 Shouldn't be a big PR, I'm just trying to force myself to make smaller PR's so that we don't have another dreaded gate_detect PR like last year. If you have time it would be great if you could look over it, if not I'm fairly confident it should be alright.

**Comment by newtoncn** [comment]
Yeah that's actually a much better idea, I was trying to think of a less intrusive way and that sounds like the one! Thanks! I'll fix that tomorrow

**Comment by newtoncn** [comment]
@garethellis0 Should be good for another review when you have time

**Comment by newtoncn** [comment]
[Make code runnable on Simulator](https://trello.com/c/ptgF3Pbc/15-make-code-runnable-on-simulator)

File: src/decision/include/OdomToTwistStamped.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Created By: Cameron Newton
+ * Created On: Sept 12nd, 2018
+ * Description: A node that remaps the output of the decision node from an Odom
+ * message
+ *              to a TwistStamped for the simulator
```
Comment (garethellis0) [inline src/decision/include/OdomToTwistStamped.h]: Looks like `clang-format` goofed up the formatting here.

File: src/decision/include/OdomToTwistStamped.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+#ifndef JORMUNGANDR_ODOMTOTWISTSTAMPED_H
+#define JORMUNGANDR_ODOMTOTWISTSTAMPED_H
```
Comment (garethellis0) [inline src/decision/include/OdomToTwistStamped.h]: This is just a general comment, but you can replace these with `#pragma once` (and get rid of the `#endif` at the end of the file) and it'll do the same thing. Not blocking this PR, but just so ya know (I only learned about this recently).

File: src/decision/src/odom_to_twist_stamped/odom_to_twist_stamped.cpp [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Created By: Cameron Newton
+ * Created On: Sept 12nd, 2018
+ * Description: A node that remaps the output of the decision node from an Odom
+ * message
+ *              to a TwistStamped for the simulator
```
Comment (garethellis0) [inline src/decision/src/odom_to_twist_stamped/odom_to_twist_stamped.cpp]: See comment above about `clang-format` messing up this comment.

File: src/decision/include/OdomToTwistStamped.h (hunk 2 above)
Comment (newtoncn) [reply src/decision/include/OdomToTwistStamped.h]: Cool, I'll check that out in the future. TBH I don't really understand what these #ifndef markers mean

**Pull request merged**